pylintrc

```diff
@@ -53,6 +53,7 @@ load-plugins=
 #
 # CHANGED:
 disable=
+    bad-option-value,
```

Comment:
Why did you add this?

Reply:
see #277. I'll remove it again

Reply:
Actually keep it. Just wanted to know why it was added 😊.